src/tests/ftest/launch.py

```diff
@@ -1019,8 +1019,7 @@ def run(self, args):
 
         # Determine if bullseye code coverage collection is enabled
         logger.debug("Checking for bullseye code coverage configuration")
-        # pylint: disable=unsupported-binary-operation
-        self.bullseye_hosts = args.test_servers | NodeSet(get_local_host())
+        self.bullseye_hosts = args.test_servers | get_local_host()
```

Comment:
I think @phender asked you to change this, but `get_local_host()` doesn't return a NodeSet on this branch (yet) so this is wonrg and should be `args.test_servers | NodeSet(get_local_host())`
Thoughts @phender ?

Reply:
`get_local_host()` does return a NodeSet: https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/host_utils.py#L67

This change was landed the other day.

Reply:
I just tries this and there's a conflict on this exact code-block.